### PR TITLE
Currency error instead of unit

### PR DESCRIPTION
**Pull request by tikhono**

### CODE REVIEW

File: primitives/src/currency.rs [hunk 1]
```diff
@@ -12,7 +12,18 @@ use scale_info::TypeInfo;
...
+#[derive(Default, Debug)]
+pub struct CurrencyConversionError {}
+
+#[cfg(feature = "std")]
+impl std::error::Error for CurrencyConversionError {}
+
+#[cfg(feature = "std")]
+impl std::fmt::Display for CurrencyConversionError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "CurrencyConversionError is here!")
```
Comment (borispovod) [inline primitives/src/currency.rs:24]: ? :) 
let's write conscious error

Comment (borispovod) [reply primitives/src/currency.rs:24]: e.g.: ("can't convert {} to currency", symbol)

Comment (borispovod) [reply primitives/src/currency.rs:24]: or ("unknown currency {}", symbol)

**Review by borispovod — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,8 @@
 .DS_Store
 
+# Intellij Idea files
+.idea/
```
Comment (fzzr-) [inline .gitignore:4]: Imho that's should be in your global ignore-config.

**Review by fzzr- — APPROVED** [review]

**Review by borispovod — APPROVED** [review]

**Pull request merged**